src/api/vdom.js

```diff
@@ -78,9 +77,9 @@ attributes.ref = function (elem, name, value) {
 attributes.skip = function (elem, name, value) {
   if (value) {
     skip();
-    elem[$skipCurrentElement] = true;
+    elem.__skip = true;
   } else {
-    delete elem[$skipCurrentElement];
+    delete elem.__skip;
```

Comment:
Wouldn't it be better to just remove the `__skip` on the way out of `elementClose`?